src/open_company/api/entry.clj

```diff
@@ -0,0 +1,50 @@
+(ns open-company.api.entry
+  (:require [compojure.core :refer (defroutes GET OPTIONS)]
+            [liberator.core :refer (defresource by-method)]
+            [open-company.api.common :as common]
+            [open-company.resources.company :as company-res]
+            [open-company.representations.common :as common-rep]
+            [open-company.representations.company :as company-rep]
+            [cheshire.core :as json]))
+
+;; ----- Representations
+;; I assume these could go into a separate representations ns but since
+;; they're just links and not full domain objects I just kept them here
+
+(defn company-link [{:keys [name] :as c}]
```

Comment:
I think I agree that `company-link` could go into `common` but the other one `links` doesn't seem to be particularly `common` or "shared" so not sure I agree about moving that as well. 

By moving we introduce another level of indirection in a place where the indirection isn't required or useful in any way. My 2 cents.